integreat_cms/cms/utils/content_translation_utils.py

```diff
@@ -49,9 +47,12 @@ def update_links_to(
         if new_content == outdated_content_translation.content:
             continue
 
-        fixed_content_translation = deepcopy(outdated_content_translation)
+        fixed_content_translation = (
```

Comment:
I think I missed some context here, why is it better to use our own implementation of `create_new_version_copy()` instead of deepcopy? :)